test/integration/include-missing-in-package-json/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.1",
   "name": "@snowpack/test-include-missing-in-package-json",
   "scripts": {
-    "testinstall": "cp -r packages/* node_modules && snowpack"
```

Comment:
Fixes inability for Windows to run test suite (which makes me wonder why our Windows tests on GitHub Actions pass 🤔 )